binding/src/main/java/se/fortnox/reactivewizard/binding/AutoBindModules.java

```diff
@@ -103,32 +107,41 @@ public void configure(Binder binder) {
             // We cache the auto bind modules in a static variable, since they will not change during execution, as they
             // are a result of the current class path.
             List<Class<? extends AutoBindModule>> autoBindModuleClasses = new ArrayList<>();
-            List<AbstractClassScanner> scanners = createScanners(bootstrapInjector);
 
-            ClassGraph classGraph      = new ClassGraph()
-                .blacklistPackages(PACKAGE_BLACKLIST)
-                .enableMethodInfo()
-                .enableAnnotationInfo();
-            try (ScanResult scanResult = classGraph.scan()) {
-                ClassScanner classScanner = new ClassScannerImpl(scanResult);
+            try (ClassScanner classScanner = getClassScanner()) {
+                List<AbstractClassScanner> scanners = createScanners(classScanner, bootstrapInjector);
                 scanners.forEach(scanner -> scanner.visit(classScanner));
-                scanResult.getClassesImplementing(AutoBindModule.class.getName()).stream()
-                    .map(classInfo -> classInfo.loadClass(AutoBindModule.class))
-                    .forEach(autoBindModuleClasses::add);
+                classScanner.findClassesImplementing(AutoBindModule.class).forEach(autoBindModuleClasses::add);
             }
             autoBindModules = autoBindModuleClasses.stream().map(bootstrapInjector::getInstance).collect(Collectors.toList());
         }
         return autoBindModules;
     }
 
-    private List<AbstractClassScanner> createScanners(Injector factoryInjector) {
-        ClassGraph classGraph = new ClassGraph().whitelistPackages(AbstractClassScanner.class.getPackage().getName());
+    protected ClassScanner getClassScanner() {
+        ClassGraph classGraph      = new ClassGraph()
+                .blacklistPackages(getPackageBlacklist())
```

Comment:
Because any classes that can be added by a developer will contribute with classes (packages) on the classpath of the fatjar, rather than with new jars.